policyengine_canada/parameters/gov/provinces/nt/tax/income/benefits/child_benefit/income_threshold/lower.yaml

```diff
@@ -0,0 +1,11 @@
+description: Northwest Territories allows families above this income threshold to receive the child benefit.
+values:
+  2022-07-01: 30_000
+metadata:
+  unit: currency-CAD
+  label: Northwest Territories child benefit lower income threshold
+  reference:
+  - title: Government of Canada - Child benefit
+    href: https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/t4114/canada-child-benefit.html#toc18
+  - title: Northwest Territories - Income Tax Act - Part 1, Division A, 3.2, (2)
+    href: https://www.justice.gov.nt.ca/en/files/legislation/income-tax/income-tax.a.pdf?t1621265645728
```

Comment:
missed
  - title: Northwest Territories - NWT Child Benefit
    href: https://www.ece.gov.nt.ca/en/isp/nwt-child-benefit#:~:text=To%20be%20eligible%20for%20all,than%20%2480%2C000%20per%20year%3B%20and